packages/calcite-components/src/custom-theme/accordionItem.ts

```diff
@@ -0,0 +1,7 @@
+import { placeholderImage } from "../../.storybook/placeholderImage";
```

Comment:
This file also needs to be kebab-cased.

Reply:
this was an old file so making this change added 8 other files to the commit but it's good cleanup.